frontend/src/components/CategoryButton.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components/macro";
+import Colors from "../constants/Colors";
+
+export default function CategoryButton({ id, children, onClick }) {
+  return <Wrapper onClick={() => onClick(id)}>{children}</Wrapper>;
```

Comment:
```suggestion
  return <Button onClick={() => onClick(id)}>{children}</Button>;
```
`Wrapper` is not a great component name, you could rather use `Button`, for instance.

Reply:
Changed it for all thank you 🍀